sdk/deviceupdate/iot-device-update-rest/review/iot-device-update.api.md

```diff
@@ -1984,14 +1963,14 @@ export interface LogCollectionOperationDeviceStatusOutput {
     status: "NotStarted" | "Running" | "Succeeded" | "Failed";
 }
 
-// @public (undocumented)
+// @public
 export interface LogCollectionOutput {
-    createdDateTime?: string;
+    readonly createdDateTime?: string;
```

Comment:
is this change from typespec?

Reply:
https://github.com/kazrael2119/azure-rest-api-specs/blob/387f63b16ff6f8010b1fffa3860fc44ee72a6be9/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json#L4037-L4038